android/src/main/java/com/revenuecat/purchases/common/common.kt

```diff
@@ -23,30 +23,6 @@ import com.revenuecat.purchases.restorePurchasesWith
 import org.json.JSONObject
 import java.net.URL
 
-fun addAttributionData(
```

Comment:
I extracted all these functions to https://github.com/RevenueCat/purchases-hybrid-common/pull/51/files#diff-714925bb39d7d0a439e0cb95f477c5db